invideoquiz/public/js/src/invideoquiz.js

```diff
@@ -50,6 +57,25 @@ function InVideoQuizXBlock(runtime, element) {
         }
     }
 
+    function getVideoDimensions() {
+        videoPosition = $('.tc-wrapper', video).position().top;
+        videoHeight = $('.tc-wrapper', video).css('height');
+        videoWidth = $('.tc-wrapper', video).css('width');
+        return [videoPosition, videoHeight, videoWidth];
+    }
+
+    function videoDimensionsDiffer(newMeasurement, oldMeasurement) {
+        if (newMeasurement.length !== oldMeasurement.length) {
```

Comment:
This case would never occur, since knownVideoDimensions is set right at the outset. Regardless, it's now unnecessary with the changes you've suggested since!